### ISSUE TITLE
Build Problem on hugoThemes

### ISSUE THREAD

**Issue by kc0bfv** [kc0bfv/ticky_tacky_dark#2]
For discussion of the build problem on hugoThemes.

**Comment by kc0bfv** [comment]
https://github.com/gohugoio/hugoThemes/issues/867

@digitalCraftsman, I'm unable to reproduce this one locally - I think this is a problem inside Hugo, based on other similar bugs I see.

Per the build log (https://app.netlify.com/sites/hugothemes/deploys/5f23f6d14febd6790c5a917e) you're using:

Hugo Static Site Generator v0.74.3-DA0437B4 linux/amd64 BuildDate: 2020-07-23T16:22:34Z

I'm using:

Hugo Static Site Generator v0.74.3/extended linux/amd64 BuildDate: 2020-07-23T20:39:21Z

When I use the hugoThemes build script, after git cloning my theme repo, it completes with no error:

```
 ==== SKIPPING  hugo-journal  (Submodule Empty) ======                                                                                                                                
 ==== PROCESSING  ticky_tacky_dark  ======                                                                                                                                            
ticky_tacky_dark is whitelisted                                                                                                                                                       
Building site for theme ticky_tacky_dark using its own exampleSite to ../themeSite/static/theme/ticky_tacky_dark/                                                                     
 ==== SKIPPING  hugo-theme-basic  (Submodule Empty) ======
```

I don't see a lot of difference between our hugo versions so I'm at a bit of a loss.  I wonder if this was just a one time fluke?  Is there some other way I can try to replicate your setup?

**Comment by digitalcraftsman** [comment]
Hi Karl,

> Is there some other way I can try to replicate your setup?

Data races can be hard to replicate. I've looked at earlier build logs in this is the first instance where I noticed the error message with your theme. Based on the log information and kind of error it might be related to  https://github.com/gohugoio/hugo/issues/7230.

I don't think there's much on our end that could fix the issue. Therefore, I would remove your theme from gohugoio/hugoThemes#867. However, it's still possible that your theme's demo can't be generated depending on the occurence of the data race. Newer build logs don't log issues with your theme.
